mqtt/mqtt-edgehub/src/tls.rs

```diff
@@ -0,0 +1,125 @@
+use openssl::{pkcs12::Pkcs12, pkey::PKey, stack::Stack, x509::X509};
+
+pub struct Identity(Vec<u8>);
```

Comment:
nit: I feel that it can be confused with iot hub identity cert. Should we call it ServerIdentity or ServerCertificate? or at least add some comments to clarify

Reply:
it lives in tls module, still confusing?